merkle/crdt/composite.go

```diff
@@ -71,15 +71,11 @@ func (m *MerkleCompositeDAG) Delete(
 }
 
 // Set sets the values of CompositeDAG. The value is always the object from the mutation operations.
-func (m *MerkleCompositeDAG) Set(
-	ctx context.Context,
-	patch []byte,
-	links []core.DAGLink,
-) (ipld.Node, uint64, error) {
+func (m *MerkleCompositeDAG) Save(ctx context.Context, data any) (ipld.Node, uint64, error) {
 	// Set() call on underlying CompositeDAG CRDT
 	// persist/publish delta
 	log.Debug(ctx, "Applying delta-mutator 'Set' on CompositeDAG")
-	delta := m.reg.Set(patch, links)
+	delta := m.reg.Set(data.([]core.DAGLink))
```

Comment:
suggestion: maybe here it also makes sense to add a type assertion.

Reply:
Yes. I wanted to get feedback before cleaning everything up.